include/wx/generic/gridsel.h

```diff
@@ -139,6 +147,9 @@ class WXDLLIMPEXP_CORE wxGridSelection
     void MergeOrAddBlock(wxGridBlockCoordsVector& blocks,
                          const wxGridBlockCoords& block);
 
+    // Called each time the selection changed or scrolled to recompute m_polyPolygon.
+    void ComputePolyPolygon(bool refreshLabelWindows = false, const wxRect& renderExtent = {});
```

Comment:
Sorry, I still think that this function shouldn't have `refreshLabelWindows` argument and that the case of `refreshLabelWindows = true` should be handled in the single place where it's used. I realize that this means we're going to potentially call `Refresh()` twice but I don't think this is a problem in practice, the refresh rectangles should be coalesced anyhow.

And it would make this function much easier to understand.

Reply:
Done

Reply:
`ComputePolyPolygon()` has been renamed to `ComputeSelectionShape()`. Because the new name seems strange to me now! I'm thinking of renaming it to `UpdateSelectionShape()` instead, what do you think ?

Reply:
Both `Compute` and `Update` seem fine to me, with maybe a slight preference for the latter, but mostly I just think that `SelectionShape()` is much better than `PolyPolygon` here.

Thanks!